src/applications/edu-benefits/10203/content/benefitSelection.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
are these different from what is in https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/utils/labels.jsx ?

Reply:
Yes, it only has the labels required for the 10203

Reply:
Instead of duplicating this file with minor changes you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete in the page to remove the property that is not needed.